string DEVICE = device
string APPLICATION = application
